NuGet.config

```diff
@@ -19,5 +19,6 @@
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="applicationinsights" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
     <add key="dotnet" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" protocolVersion="3" />
+    <add key="aspnetmodule" value="\\ddfiles\Team\Public\Correlation\AI"/>    
```

Comment:
I'm waiting for the package from the AspNet team. Can we temporary put it to AI myget once i have it? It depends on DiagSource too, so they can't publish it to nuget until DiagSource is published (i'm working on it)